unittests/Feature/BinaryFeature.cpp

```diff
@@ -44,9 +42,7 @@ TEST(BinaryFeature, BinaryFeatureChildren) {
   EXPECT_EQ(
       std::distance(FM->getFeature("a")->begin(), FM->getFeature("a")->end()),
       1);
-  if (auto *F =
-          llvm::dyn_cast<vara::feature::Feature>(*FM->getFeature("a")->begin());
-      F) {
+  if (auto *F = llvm::dyn_cast<Feature>(*FM->getFeature("a")->begin()); F) {
```

Comment:
```suggestion
  if (auto *F = llvm::dyn_cast<Feature>(*FM->getFeature("a")->begin())) {
```
If you don't have a more complex check of need the variable defined in the else branch you can just use an normal if here without initializer.